types/components/meshes/index.d.ts

```diff
@@ -1,6 +1,20 @@
 export * from './Box';
 export * from './Cone';
+export * from './Cylinder';
+export * from './Group';
+export {Dodecahedron} from './Dodecahedron';
+export * from './Extrude';
+export {Icosahedron} from './Icosahedron';
```

Comment:
What is special with `Dodecahedron` and `Icosahederon` so they are exported different way ?